lib/cisco_node_utils/bgp_af.rb

```diff
@@ -264,8 +258,7 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      result = config_get('bgp_af', 'dampen_igp_metric', @get_args)
-      result ? result.first.to_i : nil
+      config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
This raises a related question for me: why the need to specify `auto_default: false` at all? Why not use the presence of an explicit `default_value:` entry to know when to override `auto_default:`?